pkg/pillar/utils/logging/log.go

```diff
@@ -32,3 +34,33 @@ func GetMyStack() string {
 	}
 	return output
 }
+
+// NoSuitableAddrStr - the string to be used for checking the error message
```

Comment:
How about adding a comment that this should really be exported by the standard net package but isn't so we end up with a duplicate definition here.